configs/nextflow.hopper.config

```diff
@@ -41,14 +42,14 @@ process {
 		container = "${params.containerDir}/abritamr.sif"
 		cpus = 4
 		memory = '10 GB'
-		publishDir = [ path: "${params.outdir}/abritamr", mode: 'copy', overwrite: true ]
+		publishDir = [ path: "${params.outdir}/${params.species}/abritamr", mode: 'copy', overwrite: true ]
```

Comment:
Is there a particular reason you want output automatically divided per species?

Reply:
In this case we have the option to separate the outdir based on species. Though I'm thinking you can actually just specify that yourself using `outdir`.

Reply:
Just so the outputs are separated based on so you don't have outputs from different species in the same directory with no easy way to discern them.